docs/manual/docs/cmd/yammer/report/report-groupactivityfilecounts.md

```diff
@@ -0,0 +1,42 @@
+# yammer report groupactivityfilecounts
```

Comment:
File renamed from `groupactivityfilecounts` to `groupsactivitycounts` to match the issue description.